readmes/README.Linux.md

```diff
@@ -62,48 +62,39 @@ The following dependencies work on CentOS 7. For other CentOS versions, these st
     sudo ln -sf /usr/bin/cmake3 /usr/local/bin/cmake
     ```
 
-### For RHEL 7 and 8
+### For RHEL 8 or Rocky 8
 
 1. Install Development Tools.
 
-    - For RHEL 8: Install:
-
-        ```bash
-        sudo yum group install -y "Development Tools"
-        ```
+    ```bash
+    sudo yum group install -y "Development Tools"
+    ```
 
-    - For RHEL versions (< 8.0):
+2. Install dependencies:
 
-        ```bash
-        sudo yum-config-manager --enable rhui-REGION-rhel-server-rhscl
-        sudo yum install -y devtoolset-7-toolchain
-        ```
+    ```bash
+    wget -P ~/cloudberrydb/readmes/ https://public-artifacts.obs.cn-north-4.myhuaweicloud.com/README.Rhel-Rocky.bash
```

Comment:
We should cherry-pick the Rhel-Rocky.bash file from Greenplum upstream to the CBDB repo. It will be easily maintained.